pyproject.toml

```diff
@@ -43,9 +43,6 @@ license-files = ["LICENSE"]
 [tool.setuptools.packages]
 find = {namespaces = false}
 
-[tool.distutils.bdist_wheel]
```

Comment:
@mikemhenry - universal bdist_wheel entries are for pure Python projects that are both py2 and py3 compliant (you can see the effect when you build a wheel, it'll output "py2py3" in the name)